src/components/Task/Task.jsx

```diff
@@ -1,25 +1,47 @@
 import React from "react";
-import PropTypes from "prop-types";
 import styles from "./style.module.scss";
 import cn from "classnames";
+import { useSelector, useDispatch } from "react-redux";
+
+const selectTodoById = (state, todoId) => {
+  return state.todos.find((todo) => todo.id === todoId);
+};
+
+const TodoListItem = ({ id }) => {
+  const todo = useSelector((state) => selectTodoById(state, id));
```

Comment:
it is not good idea. You can get content in props. In such way you will use store only in upper component and reduce render from n+1 to 1 time